imops/measure.py

```diff
@@ -151,6 +151,9 @@ def center_of_mass(array: np.ndarray, num_threads: int = -1, backend: BackendLik
     --------
     >>> center = center_of_mass(np.ones((2, 2)))  # (0.5, 0.5)
     """
+    if array.dtype == 'bool':
+        array = array.astype(float)
```

Comment:
wouldn't float32/float16 be cheaper?

Reply:
There are some numeric instabilities in fp32 and sometimes it is not so accurate (rtol = 1e-2). Maybe it will be fixed in fully-cythonized `center_of_mass`